pkgs/applications/science/physics/yade/default.nix

```diff
@@ -0,0 +1,111 @@
+{ stdenv, fetchurl, fetchFromGitHub, pkgconfig, cmake, makeWrapper, python27Packages,
```

Comment:
is specifically `python27` needed? If both 2.7 and 3.x are supported, use `python` or `pythonPackages`.